CITi-UFPE/hackapride_2022
# Feature/awards font

# What I did?

- Change the font family and adjust start position

## How to test?

1. Open the terminal and go to project folder

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/awards-font`
   1. `git pull origin feature/awards-font`
   1. `yarn install` or `npm install`
   1. `code .`
   1. `yarn start` or `npm start`

1. In browser go to http://localhost:3000/<route>

1. Verify if the component or section looks like the web mockup
